packages/run-protocol/src/vaultFactory/vault.js

```diff
@@ -618,15 +621,15 @@ export const makeInnerVault = (
     // vaultSeat. Would using a separate seat clarify the accounting?
     // TODO what if there isn't anything to mint?
     const { vaultSeat } = state;
-    runMint.mintGains(harden({ RUN: toMint }), vaultSeat);
+    mint.mintGains(harden({ RUN: toMint }), vaultSeat);
```

Comment:
not ready to change the `RUN` keyword yet?

Reply:
No that's queued up in https://github.com/Agoric/agoric-sdk/issues/4800 and blocked on a design decision for what it will be.